fluentd/configs.d/output/fluentd_es_ops_config.conf

```diff
@@ -0,0 +1,17 @@
+    <store>
+      @type elasticsearch_dynamic
+      host "#{ENV['OPS_HOST']}"
+      port "#{ENV['OPS_PORT']}"
+      scheme https
+      index_name .operations.${Time.at(time).getutc.strftime(@logstash_dateformat)}
```

Comment:
Can you match the index_name pattern from the original .../output/operations.conf ?